tmva/sofie/inc/TMVA/ROperator_RNN.hxx

```diff
@@ -0,0 +1,619 @@
+#ifndef TMVA_SOFIE_ROPERATOR_RNN
+#define TMVA_SOFIE_ROPERATOR_RNN
+
+#include "TMVA/RModel.hxx"
+#include "TMVA/ROperator.hxx"
+#include "TMVA/SOFIE_common.hxx"
+
+#include <memory>
+#include <sstream>
+#include <vector>
+
+namespace TMVA {
+namespace Experimental {
+namespace SOFIE {
+
```

Comment:
It would be good to add some documentation about the class, like which type of RNN are supported and what not and some details on how the class is implemented.

I see the amount of code in the class implementing the operator is rather large.
Will it not be better to move the code in a .cxx file (ROperator_RNN.cxx) and have it in the src directory ? 
I am afraid the header file will be parsed by Cling for the dictionary and it might be better to reduce this code.

Reply:
@lmoneta  I’m getting some link errors.
ROperator_RNN is a templated C++ class so I can’t put the implementation in a source file.